packages/material-ui/src/Breadcrumbs/BreadcrumbCollapsed.test.js

```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 import { expect } from 'chai';
 import { spy } from 'sinon';
 import { fireEvent, createClientRender } from 'test/utils';
-import BreadcrumbCollapsed from './BreadcrumbCollapsed';
+import BreadcrumbCollapsed from '@material-ui/core/Breadcrumbs/BreadcrumbCollapsed';
```

Comment:
This module is private and the proposed import path invalid, so I think that we can keep the relative import 👍 

```suggestion
import BreadcrumbCollapsed from './BreadcrumbCollapsed';
```

Reply:
@oliviertassinari Thanks for that observation. I will continue with the others.